docs/reference/access-control-lists.md

```diff
@@ -60,11 +60,11 @@ Upgrading the lakeFS version will require migrating to the new ACL authorization
 
 In order to run the migration run:
 ```
-lakefs migrate auth-acl
+lakefs migrate up
 ```
 
-The command defaults to dry-run. We recommend running the script only after running the dry-run and going over the warnings. 
-To apply the migration, re-run with the  `--yes`  flag
+The command will run the migration to ACL. The migration process might adjust the current authorization policies to fit ACL, in that case the command will not make any changes, only print warnings.  
+In case of warnings to apply the migration, re-run with the  `--force`  flag
```

Comment:
is this true? I thought that if there are no warnings the migration does take place, only if there are warnings, it'll stop and require the `--force` flag.

Reply:
Yes, only in case of warning we need to re-run with force